test/Datadog.Trace.ClrProfiler.Managed.Tests/Datadog.Trace.ClrProfiler.Managed.Tests.csproj

```diff
@@ -28,4 +28,10 @@
   <ItemGroup Condition="'$(TargetFramework)' != 'net452'">
     <PackageReference Include="Microsoft.Data.SqlClient" Version="2.0.1" />
   </ItemGroup>
+
+  <ItemGroup Condition=" !$(TargetFramework.StartsWith('net4')) ">
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0" />
```

Comment:
I added these as well in my PR, but I don't remember why. Do you? See https://github.com/DataDog/dd-trace-dotnet/pull/1443#discussion_r678299277
cc @andrewlock

Reply:
Btw, there's already a group with condition `!$(TargetFramework.StartsWith('net4'))` in line 23 above, so these references could be moved there. (level of care low)

Reply:
This is unnecessary after you added it. I had to add it because our AspNetCoreDiagnosticObserver depended on some Microsoft.AspNetCore.Http types (I forgot which one, maybe `HttpContext`?), but we only compile against ref assemblies. So when we load the types at runtime in our test framework, it can't resolve the runtime type. So by adding the implementation assemblies with these package references, the type loads succeed